homestar-runtime/src/receipt.rs

```diff
@@ -38,18 +41,18 @@ const PROOF_KEY: &str = "prf";
 
 /// Receipt for [Invocation], including it's own [Cid] and a [Cid] for an [Instruction].
 ///
-/// `@See` [homestar_core::workflow::Receipt] for more info on some internal
+/// `@See` [homestar_invocation::Receipt] for more info on some internal
```

Comment:
```suggestion
/// See [homestar_invocation::Receipt] for more info on some internal
```
Weird formatting around See?